### PR TITLE
ftxdiff: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.es/osx/ftxdiff.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+> Compara las diferencias entre dos fuentes.
+> Más información: <https://developer.apple.com/fonts>.
+
+- Envía las diferencias a un archivo de texto específico:
+
+`ftxdiff --output {{ruta/al/archivo_de_fontdif.txt}} {{ruta/al/archivo_ont_1.ttc}} {{ruta/al/archivo_font_2.ttc}}`
+
+- Incluir nombres de glifos en la salida:
+
+`ftxdiff --include-nombres-de-glifos`
```
Comment (blueskyson) [inline pages.es/osx/ftxdiff.md:12]: You accidentally translated options themselves lol.
```suggestion
`fftxdiff --include-glyph-names`
```

File: pages.es/osx/ftxdiff.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Incluir nombres unicode en la salida:
+
+`ftxdiff --include-nombres-unicode`
```
Comment (blueskyson) [inline pages.es/osx/ftxdiff.md:16]:
```suggestion
`--include-unicode-names`
```

File: pages.es/osx/ftxdiff.md (hunk 1 above)
Comment (kbdharun) [reply pages.es/osx/ftxdiff.md:12]: Good catch

**Review by kbdharun — APPROVED** [review]

**Review by kant — COMMENTED** [review]
thaks ok

**Review by blueskyson — APPROVED** [review]

**Pull request merged**